stdlib/public/core/StaticPrint.swift

```diff
@@ -849,10 +849,12 @@ internal func constant_vprintf_backend(
 @_semantics("oslog.requires_constant_arguments")
 @inlinable
 @_transparent
+@_alwaysEmitIntoClient
 @_optimize(none)
 public func constant_vprintf(_ message: ConstantVPrintFMessage) {
   let formatString = message.interpolation.formatString
   let argumentClosures = message.interpolation.arguments.argumentClosures
+  if Bool(_builtinBooleanLiteral: Builtin.ifdef_PRINT_DISABLED()) { return }
```

Comment:
Maybe let me just explain why this isn't the very 1st line in the function: If I put it into the 1st line, the copy-propagation + constant-folding + dead-code-elimination doesn't like that, because it for some reason isn't able to fully eliminate the ConstantVPrintFMessage input. I haven't fully investigated why, but placing the check as the 3rd line (after all accesses to "message") seems to work as a simple workaround.